.github/workflows/php.yml

```diff
@@ -34,6 +37,11 @@ jobs:
       env:
         COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+    - name: Install dependencies
+      run: npm install
+    - name: Run ESLint
+      run: npm run eslint -- eslint/rules
```

Comment:
I think it should be separated into a different workflow to run the tests on different nodejs versions, also I don't think we should run eslint tests on different PHP versions